python/cugraph/tests/test_filter_unreachable.py

```diff
@@ -36,7 +35,7 @@
 SOURCES = [1]
 
 
-@pytest.mark.parametrize('graph_file', ['../datasets/netscience.csv'])
+@pytest.mark.parametrize('graph_file', utils.DATASETS)
```

Comment:
Should this be DATASETS_1 as in test_sssp.py

Reply:
To match what was there yes, but to move every test to using a common set then no.  
I had left some using the smaller number of test since there seemed to be issues